resources/public/api/conf/5.0/application.yaml

```diff
@@ -69,8 +69,8 @@ tags:
     description: Resources relating to an individual's historic non-FHL UK Property Income & Expenses Period Summaries
   - name: Foreign Property Income & Expenses Period Summary
     description: Resources relating to an individual's Foreign Property Income & Expenses Period Summary
-  - name: Foreign Property Cumulative Submission
-    description: Resources relating to an individual's Foreign Property Cumulative Submissions
+  - name: Foreign Property Cumulative Summary

```

Comment:
Please add `Period` to signify this is for periodic i.e. `Foreign Property Cumulative Period Summary`